Sources/XCMetricsBackendLib/Statistics/Model/DayCount.swift

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2021 Spotify AB.
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import Vapor
+import Fluent
+
+public final class DayCount: Model, Content {
+
+    public static let schema = "statistics_day_counts";
+
+    @ID(custom: "day")
+    public var id: Date?;
```

Comment:
Does this need to be an `optional`?

Reply:
Apparently, to conform to `Model`. An alternative would be to use a generated id and have another field for the day. But then we would lose the "built in" functionality (like enforcing uniqueness and the index)

Reply:
Got it, it's fine to be `optional` then.